rollup.config.js

```diff
@@ -21,17 +28,46 @@ const config = {
   preserveEntrySignatures: false,
   plugins: [
     typescript(),
+    postcss({
+      plugins: [
+        postcssUrl({
+          url: (asset) => {
+            if (!/\.ttf$/.test(asset.url)) return asset.url;
+            const distPath = path.join(process.cwd(), "esphome");
+            const distFontsPath = path.join(distPath, "fonts");
+            fs.ensureDirSync(distFontsPath);
+            const targetFontPath = path.join(
+              "esphome_dashboard/static/fonts/",
+              asset.pathname
+            );
+            fs.copySync(asset.absolutePath, targetFontPath);
+            const relativePath = path.relative(process.cwd(), targetFontPath);
+            console.log(relativePath);
+            const publicPath = "/";
+            return "/static/fonts/" + asset.pathname; // `${publicPath}${relativePath}`;
+          },
+        }),
+      ],
+    }), copy({
+      targets: [
+        { src: 'schema/*.json', dest: 'esphome_dashboard/static/schema' },
```

Comment:
Can we download this once, and then reuse that?

1. Add `schemas` to `.gitignore`
1. When rollup config is loaded and `schemas` doesn't exist, download it
2. keep this copy command

Reply:
And maybe a script to sync schemas that re-downloads it.